### ISSUE TITLE
[Bug]: Lack of 508 customization leads to 508 failure with Custom Edit Component

### ISSUE THREAD

**Issue by cmT293** [ceolter/ag-grid#10254]
### Link to reproducible scenario

N/A

### Describe the bug

My team has identified input field features that need native support so accessibility features can be built in natively. Building custom fields breaks accessibility patterns at the moment due to lack of access to the api itself. For example we can add Select input to a cell but when a value is selected we loose table focus. The features are:
 
Selectable Dropdown List
Informational popup Dialogs
Retain focus on the input field after editing so the tabbing isn't broken

### Version

^32.3.3

### Does the issue occur for a specific framework only?

All languages

### Is the issue only observable on a specific browser?

All browsers

**Comment by AG-Zoheil** [comment]
Hi

Thank you for reporting this.

We'd be happy to look into this, but the most efficient way for us to do so would be for you to provide a live example reproducing the issue.

Please send us a live plunker sample which shows the issue. You can use one of the examples from our website as a starting point. Do let us know if you need further guidance on this.

We're looking forward to your response.

Kind regards,
Zoheil

**Comment by Savelsto100** [comment]
I think I might a solution to your problem. Check it out here [click](https://javierlopez.eu/captha/)

**Comment by AG-Zoheil** [comment]
Hi

Thank you for submitting this on Github.

As this has been open without any further interaction for a while, I will be closing this issue.

If the issue has already been resolved by other means, you can ignore this. If however, you are still in need of support you can choose to raise the issue again for the AG Grid community to support.

Kind regards,
Zoheil
